### PR TITLE
make the loading screen fade in nicely to avoid a flash

### PR DESCRIPTION
**Pull request by johnduffell**
## Why are you doing this?

On fast connections the loading screen used to appear and disapper in a moment, which caused a disconcerting flash.  This adds a fade in.

|old loading screen on a fast connection with the flash |
|--------|
|flash|

|New loading screen on a slow connection |
|--------|
|slow|

|New loading screen on a fast connection (don't see the flash) |
|--------|
|fast|

@ionamckendrick @jranks123

### CODE REVIEW

File: app/views/test.scala.html [hunk 1]
```diff
@@ -94,9 +94,21 @@
         bottom: 0;
         left: 0;
         right: 0;
-        background: rgba(0, 0, 0, .6)
+        background: rgba(0, 0, 0, .6);
+        animation-name: fadeInOpacity;
+        animation-iteration-count: 1;
+        animation-timing-function: ease-in;
+        animation-duration: 2s;
```
Comment (johnduffell) [inline app/views/test.scala.html]: this duration can easily to be changed etc

**Comment by johnduffell** [comment]
oh in all the confusion I forgot to ship this one, I can do that next week

File: app/views/test.scala.html [hunk 2]
```diff
@@ -55,7 +55,31 @@
...
...st-button__divider{line-height:2rem;padding:0;position:relative;overflow:hidden;text-align:center}.stripe-payment-request-button__divider:after,.stripe-payment-request-button__divider:before{content:" ";position:absolute;top:50%;margin-left:-999em;height:.125rem;width:998em;border-top:.0625rem solid #dcdcdc}.stripe-payment-request-button__divider:after{left:auto;width:999em;margin:0 0 0 1em}.stripe-payment-request-button--hidden{display:none}.component-contributions-survey{border-bottom:.0625rem solid #dcdcdc;margin:0 -.625rem;padding:0 .625rem}.component-contributions-survey h3{font-family:Guardian Text Egyptian Web,Georgia,serif;letter-spacing:-.03125rem;font-weight:900;font-size:1.0625rem;margin:.25rem 0}.component-contributions-survey .component-contributions-survey__button{display:inline-block;margin-bottom:2.25rem;margin-top:.75rem}.component-contributions-survey .component-contributions-survey__button .button--survey{background-color:#121212;color:#fff}.component-contributions-survey .component-cta-link__text{padding-right:1.5625rem;font-size:1rem}body{min-height:100vh}.gu-content{position:relative;z-index:0;overflow:hidden}.gu-content--contribution-thankyou{background:#ffe5
```
Comment (johnduffell) [inline app/views/test.scala.html]: the actual changes are down here!  not minified to make it easier to port them later. but I have reduced the indentation to reduce the amount of redundant characters

**Comment by johnduffell** [comment]
I have force pushed a completely new branch over the top of this PR which goes before the proper inlining. I will do another PR to put the CSS in the proper places instead, but this could be enough to unblock the test for now

**Comment by ionamckendrick** [comment]
This is so much nicer!! Thanks @johnduffell

**Pull request merged**